io.github.wxmaxima_developers.wxMaxima.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.wxmaxima_developers.wxMaxima
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: wxmaxima
+rename-appdata-file: wxMaxima.appdata.xml
+rename-desktop-file: wxMaxima.desktop
+rename-icon: wxmaxima
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Network access
+  # for update checker
+  - --share=network
+  # Persist
+  - --persist=.maxima
+  # Filesystem access
+  - --filesystem=home
+  # Dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # OpenGL access
+  - --device=dri
+modules:
+  - wxMaxima.yaml
```

Comment:
I completely miss the point of moving it to separate file.

Reply:
> Are you going to include Maxima as a module in another application as well?

Actually, I also considered creating packages for `xmaxima` (Tcl) and `kayali` (Qt). I am also looking for a GUI for `sagemath`.

Reply:
I mean wxMaxima here.